planning/autoware_behavior_velocity_template_module/src/scene.hpp

```diff
@@ -21,9 +21,11 @@
 #include <utility>
 #include <vector>
 
-namespace behavior_velocity_planner
+namespace autoware::behavior_velocity_planner
 {
 using autoware_auto_planning_msgs::msg::PathWithLaneId;
+using ::behavior_velocity_planner::SceneModuleInterface;
+using ::behavior_velocity_planner::StopReason;
```

Comment:
Any reason for adding these lines? Anyways it LGTM. 
Please apply the changes @xmfcx requested too before merging.

Reply:
These two classes are in the `behavior_velocity_planner` namespace right now, whereas this package will be in `autoware::behavior_velocity_planner`, so I added these two `using` clauses so that the code in this PR can compile and also so that when the `behavior_velocity_planner` is refactored, the changes will be easier to track.